config/locales/en.yml

```diff
@@ -423,11 +423,13 @@ en:
   shared:
     movements:
       delete_reason: Are you sure to delete this comment?
-      movements: Movements
+      movements: Movements    
```

Comment:
@zarpilla can you delete extra spaces at the end?  For translations we only need en.yml the rest of the files are translated through [localeapp](https://github.com/coopdevs/timeoverflow/wiki/Translations-Localeapp-i18n) but thank you for translate it, don't remove the files, I will overwrite them on the release process.

Reply:
Done, sorry for the spaces ;)